sedaro/src/sedaro/utils.py

```diff
@@ -18,7 +18,7 @@
 def parse_urllib_response(response: HTTPResponse) -> Dict:
     '''Parses the response from urllib3.response.HTTPResponse into a dictionary'''
     try:
-        return orjson.loads(response.data)
+        return json.loads(response.data)
```

Comment:
Why was this updated?

Reply:
TLDR: my goof.

I was getting an error here and I thought this was my mistake of looking at a window, typing , but another window had focus.  I couldn't find 'orjson' anywhere else in the project. Turns out I had 'files to include' pointed toward a sub folder instead of being blank.